kotak-website/src/home/App.css

```diff
@@ -0,0 +1,34 @@
+.outer_div {
+  text-align:center;
+  position:absolute;
+  left:50%;
+  top:50%;
+  transform: translate(-50%, -50%);
+}
+
+.inner_div{
+  display:inline-block;
+  width:100%;
+}
+
+h2 {
+  font-family: "Avenir Next";
```

Comment:
The common and generic styles which apply everywhere can be moved to the base class and overwritten wherever necessary. 
```
font-family: "Avenir Next";
color: black;
```